extra-i18n/crow-translate/autobuild/defines

```diff
@@ -1,5 +1,5 @@
 PKGNAME=crow-translate
 PKGSEC=utils
-PKGDES="A simple and lightweight translator that allows to translate and say selected text using Google, Yandex and Bing translate API"
+PKGDES="A simple and lightweight translator that allows to translate and say selected text using multiple translate API"
```

Comment:
Again, keep it neutral and descriptive.

"A multi-backend online translator with pronunciation support"

Reply:
> Again, keep it neutral and descriptive.
> 
> "A multi-backend online translator with pronunciation support"

Sound great, will polish soon.